editor/editor_themes.cpp

```diff
@@ -143,7 +143,8 @@ Ref<Theme> create_editor_theme() {
 	Color light_color_1 = base_color.linear_interpolate(Color(1, 1, 1, 1), contrast);
 	Color light_color_2 = base_color.linear_interpolate(Color(1, 1, 1, 1), contrast * 1.5);
 
-	const int border_width = (border_size % 3) * EDSCALE;
+	const int optional_border_width = (border_size % 3) * EDSCALE;
+	const int border_width = optional_border_width == 0 ? EDSCALE : optional_border_width;
```

Comment:
I reused the idea of limiting the border with to a maximum of 2 by using %. Thinking of it it might be better to just add another MAX() function...
the idea with border_width is, that it always is bigger than 1. (more precisely it always is at least EDSCALE). optional_border_width can be 0 (that is what I tried to describe with 'optional')

would you prefer when the inline if statement would be used directly in the two cases where border_width is used? than the confusing name optional_border_width is not necessary.